product_average_consumption/models/product_product.py

```diff
@@ -23,36 +26,24 @@
 
 import time
 import datetime
-from openerp import models, fields, api
-from openerp.tools.float_utils import float_round
+from odoo import models, fields, api
+from odoo.tools.float_utils import float_round
 
 
 class ProductProduct(models.Model):
     _inherit = "product.product"
 
     # Columns Section
-    average_consumption = fields.Float(
-        compute='_average_consumption',
-        string='Average Consumption', multi='average_consumption')
-    displayed_average_consumption = fields.Float(
-        compute='_displayed_average_consumption',
-        string='Average Consumption')
-    total_consumption = fields.Float(
-        compute='_average_consumption',
-        string='Total Consumption', multi='average_consumption')
-    nb_days = fields.Integer(
-        compute='_average_consumption',
-        string='Number of days for the calculation',
-        multi='average_consumption',
+    average_consumption = fields.Float(compute='_average_consumption', string='Average Consumption')
+    displayed_average_consumption = fields.Float(compute='_displayed_average_consumption', string='Average Consumption')
+    total_consumption = fields.Float(compute='_average_consumption', string='Total Consumption')
+    nb_days = fields.Integer(compute='_average_consumption', string='Number of days for the calculation',
```

Comment:
This is actually breaking PEP8

You can try to define fields like this instead:
```py
average_consumption = fields.Float(
    compute='_average_consumption',
    string='Average Consumption',
)
displayed_average_consumption = fields.Float(
    compute='_displayed_average_consumption',
    string='Average Consumption',
)
```

1 attribute per line
always end with comma

this results in smaller diffs when adding/removing attributes

Reply:
Okay, will do it